woocommerce/woocommerce-gutenberg-products-block
# Add types to packages directory

PR adding types to `/packages/` directory. Co-authored by @opr @ralucaStan and me.

### How to test the changes in this Pull Request:

Changes in this PR should have no impact in the way the frontend works. Testing is limited to smoke test the Cart and Checkout blocks to verify there are no regressions.
